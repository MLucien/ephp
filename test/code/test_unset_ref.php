<?php

$a = 1;
$b =& $a;

$c = array( &$a, &$b, 1, 2, 3 );
var_dump($c);
unset($c);

var_dump($a);
var_dump($b);
var_dump($c);

