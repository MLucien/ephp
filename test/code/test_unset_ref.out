array(5) {
  [0]=>
  &int(1)
  [1]=>
  &int(1)
  [2]=>
  int(1)
  [3]=>
  int(2)
  [4]=>
  int(3)
}
int(1)
int(1)
NULL
